build/Dockerfile

```diff
@@ -0,0 +1,5 @@
+FROM alpine:3.6
+
+USER nobody
+
+ADD build/_output/bin/pravega-operator /usr/local/bin/pravega-operator
```

Comment:
We need to agree where to put any binaries we generate. I typically prefer to put them under `/opt`instead of `/usr/local`. It is a convention and only need to agree on it. For this PR, we don't necessarily need to make any change, but we should open an issue to track it.

Reply:
This Dockerfile is automatically created by the SDK and only used by the framework when running the operator locally, e.g., with `operator-sdk up local`. See the memcached sample operator: https://github.com/operator-framework/operator-sdk-samples/blob/master/memcached-operator/build/Dockerfile

Ref: https://github.com/operator-framework/operator-sdk/blob/00ef545399db0e4b8fe4474188a342369874162a/doc/migration/v0.1.0-migration-guide.md#copy-changes-to-deployment-manifests